spec/unit/installer/xcode/pods_project_generator_spec.rb

```diff
@@ -267,6 +288,46 @@ class Xcode
             @generator.project.targets.find { |t| t.name == 'CoconutLib-iOS' }.isa.should == 'PBXAggregateTarget'
           end
 
+          it 'creates and links app host with an iOS test native target' do
+            @generator.generate!
+            @generator.project.targets.count.should == 19
+            app_host_target = @generator.project.targets[17]
+            app_host_target.name.should == 'AppHost-iOS-Unit-Tests'
+            app_host_target.symbol_type.should == :application
+            app_host_target.build_configurations.each do |bc|
```

Comment:
nit you can remove these assertions since they are already in `app_host_installer_spec.rb`. You only care to assert only about connection between the test native target and the app host